crt_portal/cts_forms/urls.py

```diff
@@ -12,6 +12,7 @@
     path('view/<int:id>/response', ResponseView.as_view(), name='crt-forms-response'),
     path('view/<int:id>/print', PrintView.as_view(), name='crt-forms-print'),
     path('view/', index_view, name='crt-forms-index'),
+    path('view/updateProfile', ProfileView.as_view(), name='cts-forms-profile'),
```

Comment:
This may be a minor thing, but I suggest having a underscore or dashes for paths, rather than capitalCase because generally it is easier for the user to remember. Either `view/update-profile` or `view/update_profile`.

We don't have much precedent for url usage, save for `/privacy-policy` but let's be consistent here and go with dashes?